test/modules/test_modules_remind.py

```diff
@@ -384,6 +384,23 @@ def test_load_database_weirdo(tmpdir):
     assert ('#sopel', 'Admin', weird_message) in result[523549810]
 
 
+def test_load_database_irc_formatting(tmpdir):
+    tmpfile = tmpdir.join('remind.db')
+    formatted_message = (
+        'This message has italics, bold, underline, '
+        'strikethrough, and monospace.')
```

Comment:
I found a way to show them:

```suggestion
        'This message has \x1ditalics\x1d, \x02bold\x02, \x1funderline\x1f, '
        '\x1estrikethrough\x1e, and \x11monospace\x11.')
```

Reply:
I did this and more. Also added uses of the remaining IRC formatting characters from `sopel.formatting`'s constants, and improved a few things (such as some assertions that should probably have detailed failure messages).